paddle/scripts/submit_local.sh.in

```diff
@@ -43,6 +43,38 @@ fi
 
 export PYTHONPATH=${PWD}:${PYTHONPATH}
 
+
+# Check python lib installed or not.
+pip --help > /dev/null
+if [ $? -ne 0 ]; then
+    echo "pip should be installed to run paddle."
+    exit 1
+fi
+
+INSTALLED_VERSION=`pip freeze 2>/dev/null | grep '^paddle' | sed 's/.*==//g'`
+
+if [ -z ${INSTALLED_VERSION} ]; then
+   INSTALLED_VERSION="0.0.0"  # not installed
+fi
+cat <<EOF | python -
+from distutils.version import LooseVersion
+import sys
+if LooseVersion("${INSTALLED_VERSION}") < LooseVersion("@PADDLE_VERSION@"):
+  sys.exit(1)
+else:
+  sys.exit(0)
+EOF
+
+if [ $? -eq 1 ]; then  # Older version installed, or not installed at all
+   set -e
+   echo "First time run paddle, need to install some python dependencies."
+   BASEDIR=$(dirname "$0")
+   set -x
+   pip install ${BASEDIR}/../opt/paddle/share/wheels/*.whl
```

Comment:
The use may not have the permission to do pip install. Need to check whether it's successfully installed.